orderer/common/server/main_test.go

```diff
@@ -643,6 +643,32 @@ func TestUpdateTrustedRoots(t *testing.T) {
 	grpcServer.Listener().Close()
 }
 
+func TestRootServerCertAggregation(t *testing.T) {
```

Comment:
I was too lazy to inspect the test artefacts here - how do you ensure that dedup takes place?

Reply:
We have 2 channels: **foo** and **bar**. They both contain the same root TLS CA of **ca1** which is picked uniformly at random.
Additionally we have two "local config" TLS root CAs which are the same root CA **ca2**, also picked uniformly at random.

I update the dialer and then probe its config and ensure it contains only 2 entries - one of each root TLS CA.

Reply:
ack - thanks Yacov!